src/main/java/com/redlimerl/speedrunigt/instance/GameInstance.java

```diff
@@ -46,7 +46,13 @@ public void tryLoadWorld(String worldName) {
         if (worldFile != null) {
             this.loadWorld(worldFile.toPath());
             LOGGER.info("Loaded events world.");
-        } else { LOGGER.error("Didn't load events world."); }
+            boolean isRejoin = this.events.stream().anyMatch(event -> event.type.equals("leave_world"));
+            if (isRejoin) {
+                this.callEvents("rejoin_world");
```

Comment:
callEvents is usually called outside of GameInstance, but this was the most convenient place to add rejoin_world
Would probably be better to add a hasPreviouslyLeftWorld() to GameInstance so that isRejoin can be determined outside of GameInstance